server/src/controller/apis/comments.js

```diff
@@ -0,0 +1,18 @@
+import { Router } from 'express';
+import authInstance from '../../configs/authentication';
+import { getBlog } from '../../middlewares/blogs';
+import CommentsAPI from '../../services/comments/api';
+
+const router = Router();
+
+router.get('/health', CommentsAPI.health);
+
+router.post('/create/:id', authInstance.isValidUser(), getBlog, CommentsAPI.createComment);
+
+
+router.get('/:id/:page', getBlog, CommentsAPI.getTenComments);
```

Comment:
what is page?
Try to name id better, for blog id write blogID and for comment id write commentID